src/sensor/ping.cpp

```diff
@@ -504,6 +489,35 @@ void Ping::request(int id)
 
 void Ping::setLastPingConfiguration()
 {
+    static int lastSrcId = -1;
+    if(lastSrcId == _srcId) {
+        return;
+    }
+    lastSrcId = _srcId;
+
+    if(!link()->isWritable()) {
+        qCDebug(PING_PROTOCOL_PING) << "It's not possible to set last configuration when link is writable.";
+        return;
+    }
+
+    // Load last configuration
+    static bool once = false;
+    if(!once) {
```

Comment:
I think this condition should not be here anymore with the lastSrcId check above.

Reply:
I did it, my I have done some ctrl+z and this got lost.